### ISSUE TITLE
Cantera-dev is outdated

### ISSUE THREAD

**Issue by ischoegl** [Cantera/conda-recipes#24]
While 2.5.1 is now officially released, - :tada:, - the `dev` conda variant appears to be still on 2.5.0.rc1
```
conda create -n cantera-test scons numpy cython ruamel_yaml libboost h5py ipython
conda install -c cantera/label/dev cantera
conda install -c cantera/label/dev libcantera-devel
```
and
```
In [1]: import cantera as ct

In [2]: ct.__version__
Out[2]: '2.5.0rc1'
```

**Comment by bryanwweber** [comment]
I pushed a commit to `main` and this should be resolved now.

**Comment by ischoegl** [comment]
@bryanwweber ... fixed, thank you!!

**Comment by speth** [comment]
It looks like pushing 2.5.1 to the `dev` label deleted it from the `main` label. Maybe we should push `2.5.1` back to main, and now that there is a bump to the next alpha version, we have something different that can be pushed to `dev`?

**Comment by bryanwweber** [comment]
I relabeled all the existing 2.5.1 packages on anaconda.org to be in the `main` label, and triggered a new build on the `main` branch here. That should resolve the problem, I think.

**Comment by bryanwweber** [comment]
FYI @speth looks like the deprecation removals created a problem: https://github.com/Cantera/conda-recipes/runs/1927627896#step:10:1212

**Comment by ischoegl** [comment]
Should be fixed by Cantera/cantera#980

**Comment by bryanwweber** [comment]
This seems to be fixed now.
